README.md

```diff
@@ -24,7 +24,7 @@ This project defines core build rules for [Scala](https://www.scala-lang.org/) t
 * [scala_test_suite](docs/scala_test_suite.md)
 * [thrift_library](docs/thrift_library.md)
 * [scala_proto_library](docs/scala_proto_library.md)
-* [scala_toolchain](docs/scala_toolchain.md)
+* [scala_](docs/scala_toolchain.md)
```

Comment:
I believe it's supposed to be `scala_toolchain`

Reply:
oops